reference-architecture/vmware-ansible/roles/rhn-subscription/tasks/main.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Is the host already registered?
+  command: "subscription-manager list"
+  register: subscribed
+  ignore_errors: yes
+
+- name: Register host via Activation key
+  redhat_subscription:
+    activationkey: "{{ rhsm_activation_key }}"
+    org_id: "{{ rhsm_org_id }}"
+    state: present
+    pool: "^(60 Day Supported OpenShift Enterprise|OpenShift Enterprise, Standard|OpenShift Enterprise, Premium|Employee)"
```

Comment:
@cooktheryan Just a heads up. Your readme.md in that pull has a typo in it. 

+./ose-on-aws.py --create-vpc=no --byo-bastion=yes --keypair=OSE-key --rhsm-user=rh-user --rhsm-password=password --public-hosted-zone=sysdeseng.com --rhsm-pool="Red Hat OpenShift Container Platform, Standard, 2-Core --bastion-sg=sg-a32fa3"

You are missing your closed " on your pool id